packages/base-driver/test/e2e/express/server.e2e.spec.js

```diff
@@ -120,32 +172,49 @@ describe('server plugins', function () {
       httpServer[`_updated_${route}`] = true;
     };
   }
-
-  it('should allow one or more plugins to update the server', async function () {
-    hwServer = await server({
-      routeConfiguringFunction: _.noop,
-      port,
-      serverUpdaters: [
-        updaterWithGetRoute('plugin1', 'res from plugin1 route'),
-        updaterWithGetRoute('plugin2', 'res from plugin2 route'),
-      ],
-    });
-    let {data} = await axios.get(`http://${TEST_HOST}:${port}/plugin1`);
-    data.should.eql('res from plugin1 route');
-    ({data} = await axios.get(`http://${TEST_HOST}:${port}/plugin2`));
-    data.should.eql('res from plugin2 route');
-    hwServer._updated_plugin1.should.be.true;
-    hwServer._updated_plugin2.should.be.true;
-  });
-  it('should pass on errors from the plugin updateServer method', async function () {
-    await server({
-      routeConfiguringFunction: _.noop,
-      port,
-      serverUpdaters: [
-        () => {
-          throw new Error('ugh');
-        },
-      ],
-    }).should.eventually.be.rejectedWith(/ugh/);
-  });
+  given(givens).it(
+    'should allow one or more plugins to update the server',
+    async function ({protocol}) {
+      let correctPort;
+      if (protocol === 'http') {
+        correctPort = httpPort;
+        httpHwServer = await server({
+          routeConfiguringFunction: _.noop,
+          port: correctPort,
+          serverUpdaters: [
+            updaterWithGetRoute('plugin1', 'res from plugin1 route'),
+            updaterWithGetRoute('plugin2', 'res from plugin2 route'),
+          ],
+        });
+      } else {
+        correctPort = httpsPort;
+        httpsHwServer = await startHttpsServer(httpsPort, _.noop, [
+          updaterWithGetRoute('plugin1', 'res from plugin1 route'),
+          updaterWithGetRoute('plugin2', 'res from plugin2 route'),
+        ]);
+      }
+      let {data} = await axios.get(`${protocol}://${TEST_HOST}:${correctPort}/plugin1`);
+      data.should.eql('res from plugin1 route');
```

Comment:
use expressions like this instead (here and elsewhere):

```suggestion
      await axios.get(`${protocol}://${TEST_HOST}:${correctPort}/plugin1`).should.eventually.eql({data: 'res from plugin1 route'});
```